src/Components/CardList.tsx

```diff
@@ -297,10 +298,10 @@ export const CardList = ({
             totalPages={totalPages}
             isFirstPage={isFirstPage}
             isLastPage={isLastPage}
-            showFilterBtn
           />
           <List>
             {listType === 'filtered' ? <QuizEncouragement /> : null}
+            <FilterButton />
```

Comment:
I think the filter button should only be displayed on the "All" page - it's the only one that it acts on. You an use the line above to conditionally render it on that page only.

Reply:
Nice :D I've updated that